tests/integration/kubernetes/k8s-policy-job.bats

```diff
@@ -107,10 +107,9 @@ test_job_policy_error() {
         "spec.template.spec.volumes[+].name" \
         "unexpected-volume1"
 
-    yq write -i \
-        "${incorrect_yaml}" \
-        "spec.template.spec.volumes[-1].emptyDir.medium" \
-        "Memory"
+    yq -i \
+        ".spec.template.spec.volumes[-1].emptyDir.medium = \"Memory\"" \
+        "${incorrect_yaml}"
```

Comment:
nit:
I'm not sure this change belongs in the `ci: skip go version check` commit